src/cloud/benchflow/sender/sender.go

```diff
@@ -161,27 +208,17 @@ func consumeFromTopic(t TransformerSetting) {
 				}
 			fmt.Println(t.Topic+" received: "+msg.Minio_key)
 			minioKeys := strings.Split(msg.Minio_key, ",")
-			for _, k := range minioKeys {
+			containerIds := strings.Split(msg.Container_id, ",")
+			hostIds := strings.Split(msg.Host_id, ",")
```

Comment:
@VincenzoFerme From the collectors I'm sending a list of hosts, in case it's ever needed for one reason or another.